Gruntfile.js

```diff
@@ -139,11 +157,13 @@ module.exports = function (grunt) {
 
   grunt.registerTask('build-standalone', 'Build out/ (no checks, no WPT)', [
     'run:build-out',
+    'run:copy-assets',
     'run:generate-version',
     'run:generate-listings',
   ]);
   grunt.registerTask('build-wpt', 'Build out/ (no checks)', [
     'run:build-out-wpt',
+    'run:copy-assets-wpt',
```

Comment:
This has not been tested...

Reply:
Verified by running the build command and find the pictures has been copied to the out-wpt/resources